### DISCUSSION TITLE
[OKD 4.5] community-operators liveness probe failed

### DISCUSSION THREAD

**Discussion by stu93303148** [okd-project/okd#767] [Q&A]
<img width="846" alt="截圖 2021-07-14 下午5 01 57" src="https://user-images.githubusercontent.com/1113604/125594890-855367d7-4725-4c32-ab5e-902a1c3e4a89.png">

<img width="1141" alt="截圖 2021-07-14 下午5 02 29" src="https://user-images.githubusercontent.com/1113604/125594946-e6497ad0-4f7b-44bd-968a-7932dab67030.png">

**Comment by vrutkovs** [comment]
Please attach (or upload to the public file sharing service) [must-gather](https://docs.okd.io/latest/support/gathering-cluster-data.html#support_gathering_data_gathering-cluster-data) archive

Also, is it reproducible on stable OKD 4.7?

**Reply by stu93303148** [reply]
I had installed 4.7, it's good.
But now, my customer's lab installed the OKD 4.5. 
So.... I need to rebuild same version of OKD in my lab.

must-gather download link:
https://drive.google.com/file/d/1TVOPBhNJOmsJiE123ylYQK-pwcVvZ3AJ/view?usp=sharing

**Reply by vrutkovs** [reply]
I don't see anything wildly wrong in this pod, however seems etcd is struggling:

>2021-07-14T09:01:31.790858004Z 2021-07-14 09:01:31.788526 W | etcdserver: read-only range request "key:\"/kubernetes.io/pods/openshift-marketplace/community-operators-787656f85-nw24w\" " with result "range_response_count:1 size:8433" took too long (10.909454375s) to execute

Do you see any alerts firing during that time (sadly in 4.5 we don't collect those)

**Comment by ksingh7** [comment]
This is the same issue I encountered today with ODK 4.5 

@stu93303148  did you manged to slove this ? 

![image](https://user-images.githubusercontent.com/9701902/132258382-48ee6d66-136f-44f4-919b-55d85617b406.png)

**Reply by ksingh7** [reply]
Okay a temporary fix is the to remove readiness and liveliness probes sections from the Deployment config.
